### PR TITLE
Retrieve

### PR DESCRIPTION
**Pull request by Incorrect-password**
retrieve data from database and var_dumped

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+$db = new PDO('mysql:host=db;dbname=Collection','root','password');
+
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$collection = $db->query('SELECT `latin_name`,`common_name`,`height(cm)`,`cap_width(cm)`,`deadly` FROM `Collection`');
```
Comment (georginaw) [inline index.php:6]: Is your table called 'Collection' or is that the database name? The error I get when I run the code is: Fatal error: Uncaught Error: Call to a member function fetchAll() on boolean in /var/www/html/Collection/index.php:8 Stack trace: #0 {main} thrown in /var/www/html/Collection/index.php on line 8

I had this issue...

File: test/index.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?php
```
Comment (georginaw) [inline test/index.php:1]: Is this the unit tests doc?

**Review by georginaw — COMMENTED** [review]
Reviewed

File: test/index.php (hunk 2 above)
Comment (Incorrect-password) [reply test/index.php:1]: yes but its supposed to be empty

**Pull request closed without merging**